test/actions/katello/host/attach_subscriptions_test.rb

```diff
@@ -33,5 +33,22 @@ class AttachSubscriptionsTest < ActiveSupport::TestCase
                                           :quantity => 2, :pool_uuid => @pool.cp_id
       end
     end
+
+    describe 'attach subscriptions failure' do
+      let(:action_class) { ::Actions::Katello::Host::AttachSubscriptions }
```

Comment:
This `let` is duplicated between this test and the test above. you can move it to the outer block to be shared by both :)